test_scripts/Defects/6_1/3136_01_Proprietary.lua

```diff
@@ -0,0 +1,148 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3136
+--
+-- Description: Successful 2nd HMI PTU if it's triggered within failed retry for the 1st PTU
```

Comment:
I think that `HMI PTU` should not be mentioned in scripts for develop.

Reply:
@GetmanetsIrina Please find update in 0e0c08b